autocertkit/tests/xenapi_mock.py

```diff
@@ -376,19 +440,32 @@ def get_management_interface(self, opaque):
         return self.__getHost(opaque).PIFs[0].opaque
 
     def call_plugin(self, host_ref, plugin, method, *arg):
+        if self.session.fail_plugin:
+            m = mock.Mock(side_effect=XenAPI.Failure(
+                "raised by Mock(): plugin failed"))
+            m()
+
         if plugin == "autocertkit":
```

Comment:
@bpwook There is a problem with this design, I want to pass host_ref to call_plugin so that we can specify host on which plugin code will run. If I use `obj = self.supportedPlugins[plugin]` then i am loosing way to specify host_ref or in other terms not telling where to run plugin - on master or  on slave?